### PR TITLE
Remove url params from sidebar nav

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
#122 

Similar to what we did in skyux2-docs: https://github.com/blackbaud/skyux2-docs/pull/1002/files#diff-e9cfae0087847708878c955c0e309933R98

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-nav-fix](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-nav-fix)

File: src/app/public/modules/demo-page/demo-page.component.ts [hunk 1]
```diff
@@ -155,7 +155,8 @@ export class SkyDocsDemoPageComponent implements OnInit, AfterContentInit, After
           children: results.map((component: SkyDocsComponentInfo) => ({
             name: component.name,
             // Replace host + SPA so Stache will mark active
-            path: component.url.replace(currentHostUrl, '')
+            // Remove URL params, as they only contain svcid params which are automatically handled by the nav service.
+            path: component.url.replace(currentHostUrl, '').split('?')[0]
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/demo-page/demo-page.component.ts:159]: Is there a spec for demo-page? If not, we should probably add one.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-nav-fix](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-nav-fix)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-nav-fix](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-nav-fix)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**